dbt_copilot_helper/commands/conduit.py

```diff
@@ -277,34 +276,52 @@ def start_conduit(app: str, env: str, addon_type: str, addon_name: str = None):
 
 
 @click.command(cls=ClickDocOptCommand)
-@click.argument("addon_type", type=click.Choice(CONDUIT_ADDON_TYPES))
 @click.option("--app", help="AWS application name", required=True)
 @click.option("--env", help="AWS environment name", required=True)
 @click.option("--addon-name", help="Name of custom addon", required=True)
```

Comment:
Addon name as an option doesn't quite feel right. Maybe it will look nicer as an argument.

`copilot-helper conduit <addon-name> --app <app> --env <env>`